src/main/java/org/terasology/physicalstats/component/IPhysicalStatsModifyEffect.java

```diff
@@ -17,6 +17,21 @@
 
 import org.terasology.entitySystem.entity.EntityRef;
 
+/**
+ * Interface for the application of physical stat modifiers over a period of time.
+ */
 public interface IPhysicalStatsModifyEffect {
-    void applyMod(EntityRef instigator, EntityRef entity, PhysicalStatsModifier magnitude, long duration);
+    /**
+     * Apply a PhysicalStatsModifier on the entity for the given duration.
+     *
+     * @param instigator    The instigator who is applying this modifier on the entity. It can be another entity, block,
+     *                      item, etc.
+     * @param entity        The entity who the physical stats modifier is being applied on.
+     * @param magnitude     The physical stats modifier that's going to be applied to the entity.
+     * @param duration      The duration of the effect in milliseconds. If this is less than 0, the effect will be
```

Comment:
I'm not catching most the module changes these days, but just spotted this by chance: `IPhysicalStatsModifyEffect` - we don't generally prefix interfaces/implementations and such (the `I`), relying on IDE markup instead of naming :-)